pages/common/calligrasheets.md

```diff
@@ -0,0 +1,16 @@
+# calligrasheets
+
+> Calligra's spreadsheet application.
+> More information: <https://manned.org/calligrasheets>.
+
+- Launch the spreadsheet application:
+
+`calligrasheets`
+
+- Open a specific spreadsheet:
+
+`calligrasheets {{path/to/spreadsheet}}`
+
+- Display a help or version:
+
+`calligrasheets --{{help|version}}`
```

Comment:
Same here, `a` won't come here or you can make it into separate commands. This applies to other pages too.